src/main/java/jsr223/perl/PerlScriptEngineFactory.java

```diff
@@ -25,50 +25,38 @@
  */
 package jsr223.perl;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineFactory;
 
 import jsr223.perl.utils.PerlVersionGetter;
-import processbuilder.PerlSingletonPerlProcessBuilderFactory;
-import processbuilder.utils.PerlProcessBuilderUtilities;
 
 
 public class PerlScriptEngineFactory implements ScriptEngineFactory {
-    private final Map<String, Object> parameters = new HashMap<>();
-
-    private static PerlProcessBuilderUtilities processBuilderUtilities = new PerlProcessBuilderUtilities();
-
-    private static PerlVersionGetter perlVersionGetter = new PerlVersionGetter(processBuilderUtilities);
+    private static final Map<String, Object> parameters = new HashMap();
 
     // Script engine parameters
     private static final String NAME = "perl";
```

Comment:
The 5 class variables seem useless since their value is already stored in `parameters`. Please remove them and use `parameters` when needed.